bimmer_connected/state.py

```diff
@@ -66,11 +66,11 @@ def backend_parameter(func):
     def _func_wrapper(self: 'VehicleState', *args, **kwargs):
         # pylint: disable=protected-access
         if self._attributes is None:
-            raise ValueError('No data available!')
+            raise ValueError('No data available for vehicle status!')
         try:
             return func(self, *args, **kwargs)
         except KeyError:
```

Comment:
Can the `AttributeError` from this [issue](https://github.com/m1n3rva/bimmer_connected/issues/69#issuecomment-386913701) be added?

Reply:
It can, but it won't fix the issue #69...

--> merging the change now, we can add more things later on.